test/e2e/reboot.go

```diff
@@ -126,8 +115,7 @@ func rebootNode(c *client.Client, provider, name string, result chan bool) {
 	}
 
 	// Get all the pods on the node.
-	podList, err := c.Pods(api.NamespaceDefault).List(
-		labels.Everything(), fields.OneTermEqualSelector(client.PodHost, name))
+	podList, err := listPods(c, api.NamespaceDefault, labels.Everything(), fields.OneTermEqualSelector(client.PodHost, name))
```

Comment:
May as well do this now, before merge.
You can use @lavalamp 's new framework as per example in #8747
f := NewFramework("reboot")